internal/core/ebook.go

```diff
@@ -252,3 +257,24 @@ func GetImages(html string) (map[string]string, error) {
 
 	return images, nil
 }
+
+func GenerateEPUBIdentifier() string {
```

Comment:
This method does not generate a valid UUID. You can use `github.com/google/uuid` for that. Maybe we can generate the UUID from the source URL? `uuid.FromBytes()` should help you here.